data/map.txt

```diff
@@ -29036,6 +29085,11 @@ planet Shiver
 		threshold 2000
 		fleet "Small Republic" 10
 
+planet "Builder Settlement"
+	attributes ka'het uninhabited "requires: ka'sei"
+	landscape land/asteroid0
+	description `This asteroid has seen hundreds of thousands of years of excavation made by swarms of Ka'sei. Although its size rivals that of a small moon, a large section has been hollowed out by the Builders to be filled with iridium, presumably as a form of long term storage. Due to its very low gravity, the asteroid is permanently covered in clouds of dust, orbiting it only a few meters above the surface.`
```

Comment:
> hundred of thousands

I have never seen that kind of term used. Sounds a sort of probability/sample size rather than quantity, where the original is how I've always heard it.

Reply:
Oops, sorry, that is indeed a typo on my part. You can ignore that.